bin/webpack.js

```diff
@@ -161,10 +170,16 @@ function processOptions(options) {
 	}
 
 	var firstOptions = Array.isArray(options) ? (options[0] || {}) : options;
-
-	if(typeof options.stats === "boolean" || typeof options.stats === "string") {
+	if(hasPreset(options)) {
 		var statsPresetToOptions = require("../lib/Stats.js").presetToOptions;
-		options.stats = statsPresetToOptions(options.stats);
+
+		if(Array.isArray(options)) {
+			options = options.forEach(function(singleOption) {
+				singleOption.stats = statsPresetToOptions(singleOption.stats);
+			});
+		} else {
+			options.stats = statsPresetToOptions(options.stats);
+		}
 	}
 
 	var outputOptions = Object.create(options.stats || firstOptions.stats || {});
```

Comment:
Only problem I see with that is that `Stats.presetToOptions` is really for just taking a string and converting it. I could add the array handling case but that could technically negate the truthyness of passing an empty array to `stats: []`. 

If we took it that far I could check to see if the array had a length to it to determine if it was actually an array of configs with stats strings inside, but feels like a lot of unneeded logic. 

I could maybe see creating a `Stats.optionsArrayWithStatsPresetToOptionsArray`. It would probably just iterate over the array and call `Stats.statsToOptions` on `options.stats`.